mysql-test/t/cte_recursive.test

```diff
@@ -1304,6 +1304,7 @@ drop table folks;
 create table t1(a int);
 insert into t1 values (0),(1),(2),(3),(4),(5),(6),(7),(8),(9);
  
+--source include/analyze-format.inc
```

Comment:
Ok, removed this part of the patch. Travis reported the error as per the JIRA bug but I can't see how it occurred. Will add something later if I find it.